site/src/externalContributors.json

```
@@ -82,5 +82,12 @@
     "title": "Hacktoberfest Experience",
     "link": "https://dev.to/kiran_baliga/my-hacktoberfest-2024-journey-2odn",
     "contributor": "Kiran Baliga"
+  },
+  {
+    "type": "blog",
+    "thumbnail": "/img/external-contributor/Lymah123_understanding_dwns",
```

```suggestion
    "thumbnail": "/img/external-contributor/Lymah123_understanding_dwns.png,
```
forgot the extension

---

> forgot the extension

Fixed. Thank you